bentoml/yatai/__init__.py

```diff
@@ -75,4 +84,100 @@ def get_yatai_service(
         )
 
 
-__all__ = ["get_yatai_service"]
+def start_yatai_service_grpc_server(db_url, repo_base_url, grpc_port, ui_port, with_ui):
+    track_server('yatai-service-grpc-server')
+    yatai_service = get_yatai_service(db_url=db_url, repo_base_url=repo_base_url)
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    add_YataiServicer_to_server(yatai_service, server)
+    debug_mode = config().getboolean('core', 'debug')
+    if debug_mode:
+        try:
+            logger.debug('Enabling gRPC server reflection for debugging')
+            from grpc_reflection.v1alpha import reflection
+            from bentoml.proto import yatai_service_pb2
+
+            SERVICE_NAMES = (
+                yatai_service_pb2.DESCRIPTOR.services_by_name['Yatai'].full_name,
+                reflection.SERVICE_NAME,
+            )
+            reflection.enable_server_reflection(SERVICE_NAMES, server)
+        except ImportError:
+            logger.debug(
+                'Failed enabling gRPC server reflection, missing required package: '
+                '"pip install grpcio-reflection"'
+            )
+    server.add_insecure_port(f'[::]:{grpc_port}')
+    server.start()
+    logger.info('* Starting BentoML YataiService gRPC Server')
```

Comment:
use just one log statement for this?

Reply:
One statement won't look pretty.
Multiply statements look nice like
```
2020-03-14 23:02:36,582] INFO - * Starting BentoML YataiService gRPC Server
[2020-03-14 23:02:36,583] INFO - * Debug mode: off
[2020-03-14 23:02:36,583] INFO - * Web UI: running on http://127.0.0.1:3000
[2020-03-14 23:02:36,583] INFO - * Running on 127.0.0.1:50051 (Press CTRL+C to quit)
```